src/ios/BranchSDK.m

```diff
@@ -269,7 +268,7 @@ - (void)logout:(CDVInvokedUrlCommand*)command
         }
         [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
     }];
-    self.branchUniversalObjArray = nil;
+    self.branchUniversalObjArray = [[NSMutableArray alloc] init];
```

Comment:
If you nil out the array, once you log out you won't be able to make more universal objects, so instead we make a new array.